lib/matplotlib/colors.py

```diff
@@ -2599,12 +2611,12 @@ def blend_hsv(self, rgb, intensity, hsv_max_sat=None, hsv_max_val=None,
             An (M, N, 3) RGB array of floats ranging from 0 to 1 (color image).
         intensity : `~numpy.ndarray`
             An (M, N, 1) array of floats ranging from 0 to 1 (grayscale image).
-        hsv_max_sat : number, default: 1
+        hsv_max_sat : number, optional
```

Comment:
Why not simply 
```suggestion
        hsv_max_sat : number, default: 0
```
? And likewise below.

Reply:
I thought the "rule" was that optional argument = None => default?

Reply:
Actually, it should say that it defaults to the value set upon creation.